shared/js/background/utils.es6.js

```diff
@@ -62,45 +65,45 @@ function getCurrentTab (callback) {
 }
 
 // Browser / Version detection
-// 1. Set browser for popup asset paths
-// 2. Determine if upgradeToSecure supported (firefox 59+)
-// chrome doesn't have getBrowserInfo so we'll default to chrome
-// and try to detect if this is firefox.
-
-var browser = 'chrome'
-var upgradeToSecureSupport = false
-
-try {
-    chrome.runtime.getBrowserInfo((info) => {
-        if (info.name === 'Firefox') {
-            browser = 'moz'
-
-            var browserVersion = info.version.match(/^(\d+)/)[1]
-            if (browserVersion >= 59) {
-                upgradeToSecureSupport = true
-            }
-        }
-    })
-} catch (e) {}
-
+// Get correct name for fetching UI assets
 function getBrowserName () {
```

Comment:
Keeping getBrowserName() because we need the browser name to be either 'chrome' (lowercase) or 'moz' in order to fetch the right UI assets